client_integration_test.go

```diff
@@ -0,0 +1,184 @@
+package notify_test
+
+import (
+	"os"
+	"testing"
+
+	notify "github.com/govau/notify-client-go"
+)
+
+func setup(t *testing.T) (*notify.Client, string, string) {
```

Comment:
@andrewoh531 this is a good use case for named return arguments:

```go
func setup(t *testing.T) (client *notify.Client, smsTemplateID string, emailTemplateID string) {}
```

Because at a glance it's hard to see what the two `string, string`s are.

Also could you please change `xxxId` -> `xxxID` in the local variable names? Thanks!